modules/camera/camera_picamera2.py

```diff
@@ -0,0 +1,70 @@
+"""
+Picamera2 implementation of the camera wrapper.
+"""
+
+import numpy as np
+
+try:
+    import picamera2
+except ImportError:
+    pass
+
+from . import base_camera
```

Comment:
What is the linter error?

Reply:
No longer there, I think we're good. Was telling me to place third party imports above (This is when I had the flag)